docs/project-imports/dcr.md

```diff
@@ -0,0 +1,117 @@
+# Importing `dotcom-rendering` into `csnx`
+
+## Glossary
+
+- `dotcom-rendering` refers to the application that renders HTML for `frontend`
+- _"the `dotcom-rendering` repo"_ refers to the repo that contains `dotcom-rendering`, `apps-rendering` and `common-rendering` codebases
+
+## Intention
+
+We want to relocate `dotcom-rendering` in the `csnx` monorepo.
+
+### Why?
+
+- to make it easier for Guardian product teams to ship by reducing the friction of interdependency between `dotcom-rendering` and other Guardian projects (Source, Libs, consent, commercial code etc)
```

Comment:
potentially, definitely. i don't think we have opinion yet on what happens after DCR but that's more the absence of a plan, rather than a plan to _not_ move things.

we also want to make it easier for everyone to understand how/what relocating projects needs so it's not only us who needs to pick it up (if that's useful)